### ISSUE TITLE
Header trident svg is aria-hidden but has an aria-description

### ISSUE THREAD

**Issue by mmeller75** [indiana-university/rivet-source#200]
## Describe the bug
The IU Trident in the header component is `aria-hidden`, which hides it from assistive technology, and does not have alt text, but also includes an `aria-describedby` attribute. The svg doesn't have an accessible name, so an accessible description is confusing and is flagged as an error by some automated testing tools.

## Expected behavior
Since the svg is `aria-hidden`, the description should be removed.

## Relevant context (required)
![Screen Shot 2019-09-26 at 10 56 54 AM](https://user-images.githubusercontent.com/47121512/65699705-77968400-e04c-11e9-95d5-d57525dca946.png)

## Steps to reproduce

1. Open the [header component page](https://rivet.iu.edu/components/navigation/header/) 
2. Run ARC Toolkit accessibility tester.

**Comment by illusivesunrae** [comment]
Merged into master in PR #207.
